memstore/job_manager.go

```diff
@@ -59,6 +82,9 @@ func (m *archiveJobManager) generateJobs() []Job {
 
 	now := uint32(utils.Now().Unix())
 	var jobs []Job
+	if !m.enabled {
```

Comment:
it is protected by the lock of archiveJobManager

Reply:
actually, I'm thinking about instead of duplicating the logic in each of job manager, we can put this enable/disable logic in the scheduler. e.g. keep a enable/disable map in scheduler